fluss-server/src/main/java/com/alibaba/fluss/server/kv/KvManager.java

```diff
@@ -85,6 +93,12 @@ private KvManager(
         this.arrowBufferAllocator = new RootAllocator(Long.MAX_VALUE);
         this.memorySegmentPool = LazyMemorySegmentPool.create(conf);
         this.zkClient = zkClient;
+        this.remoteKvDir = FlussPaths.remoteKvDir(conf);
+        try {
+            this.remoteFileSystem = remoteKvDir.getFileSystem();
+        } catch (Exception e) {
```

Comment:
Just throw the `IOException`, this should block the server from starting as it doesn't work if the filesystem is not initialized.

Reply:
i will update this code.